runtime/plaid/src/apis/web/mod.rs

```diff
@@ -93,31 +87,47 @@ impl Web {
             )));
         }
 
+        // TODO Discuss with @obelisk if we want to validate fields like 'iat' and 'exp'
+
+        let mut claims = HashMap::<String, Value>::new();
+        claims.insert("sub".to_string(), Value::String(request.sub.clone()));
+        claims.insert("iat".to_string(), Value::Number(request.iat.into()));
+        claims.insert("exp".to_string(), Value::Number(request.exp.into()));
+
+        // Include extra fields, but only if they are allowlisted in the config
+        if let Some(extra_fields) = request.extra_fields {
+            if let Some(allowlisted_extras) = &self.config.allowlisted_extra_fields {
```

Comment:
Would also add at the key level extra fields that are allowed in the header and body (independently).